client/package.json

```diff
@@ -17,8 +17,8 @@
     "url": "git+https://github.com/kbventures/ecommerce.git"
   },
   "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "author": "Ken Beaudin Dmitry Kulakov, Abdul Shabbir & Enrick Anderson Ong",
```

Comment:
Just a comma between your name and Dmitry's and a spelling error on my name as well. Change it to "Andersen". 
Thank you